testing/checkin/runner/mod.rs

```diff
@@ -75,13 +77,59 @@ deno_core::extension!(
     "checkin:testing" = "testing.ts",
     "checkin:timers" = "timers.ts",
     "checkin:worker" = "worker.ts",
+    "checkin:throwInExt" = "throwInExt.ts",
   ],
   state = |state| {
     state.put(TestFunctions::default());
     state.put(TestData::default());
   }
 );
 
+fn create_runtime_with_snapshot(
```

Comment:
No need to split this -- we should run all the tests with a snapshot. Let's split this change out from this PR, however and land it seperately.